DFW.h

```diff
@@ -1,18 +1,33 @@
-// Serial controller library for DFWireless controllers
+/**
+ * Serial controller library for DFWireless controllers

```

Comment:
This i disagree with. This library is not just a controller reader, it is the competition framework. Explicatly it is a framework, not a library. I talked this over with Miller, Stafford and Gennert. This model is what they want to see.